test/test_jit.py

```diff
@@ -1216,10 +1238,12 @@ def observe(x, name):
         self.assertEqual(value_stats['z'][1], x2 - y2)
 
     def test_insert_quantdequant_consecutive_qnodes_script(self):
+        x1 = torch.ones([1, 1, 1, 1])
+
         class testModule(torch.jit.ScriptModule):
             def __init__(self):
                 super(testModule, self).__init__()
-                self.conv1 = nn.Conv2d(1, 20, 5, 1)
+                self.conv1 = nn.Conv2d(1, 1, 1, 1)
```

Comment:
This seems an unrelated (good) change, could you please split it into a separate PR?